static/app/views/alerts/rules/issue/ticketRuleModal.tsx

```diff
@@ -178,16 +178,24 @@ class TicketRuleModal extends AbstractExternalIssueForm<Props, State> {
         const prevChoice = instance?.[field.name];
         // Note that field.choices is an array of tuples, where each tuple
         // contains a numeric id and string label, eg. ("10000", "EX") or ("1", "Bug")
-        if (
-          prevChoice && field.choices && Array.isArray(prevChoice)
-            ? // Multi-select fields have an array of values, eg: ['a', 'b'] so we
-              // check that every value exists in choices
-              prevChoice.every(value => field.choices?.some(tuple => tuple[0] === value))
+
+        if (!prevChoice) {
+          return field;
+        }
+
+        let should_default_choice = true;
```

Comment:
camelCase. Welcome to the frontend world! 😆

Reply:
🤦🏼‍♂️ right, thanks!